docker/config.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+UPDATE_HOSTS=${HAS_SUDO:-true}
+
+HOSTS="platform.publiq.local"
+
+if [ "$UPDATE_HOSTS" = "true" ]; then
+  MISSING_HOSTS=""
+
+  set -- $HOSTS
+  for HOST; do
+    if ! grep -q "$HOST" /etc/hosts; then
+      MISSING_HOSTS="$MISSING_HOSTS $HOST"
+    fi
+  done
+
+  set -- $MISSING_HOSTS
+  for MISSING_HOST; do
+    echo "$MISSING_HOST has to be in your hosts-file, to add you need sudo privileges"
+    sudo sh -c "echo '127.0.0.1 $MISSING_HOST' >> /etc/hosts"
```

Comment:
I think this misses a newline, because now it will be added to the last line of the file and it should be on a new line.